aws-enigma-2/main.tf

```diff
@@ -0,0 +1,205 @@
+  cyclist_version            = "v0.5.0"
+  docker_storage_dm_basesize = "19G"
+  env                        = "${var.env}"
+  env_short                  = "${var.env}"
+  github_users               = "${var.github_users}"
+  heroku_org                 = "${var.aws_heroku_org}"
+  index                      = "${var.index}"
+  registry_hostname          = "${data.terraform_remote_state.vpc.registry_hostname}"
+
+  security_groups = [
+    "${module.aws_az_1b.workers_org_security_group_id}",
+    "${module.aws_az_1b2.workers_org_security_group_id}",
+    "${module.aws_az_1e.workers_org_security_group_id}",
+    "${module.aws_az_1e2.workers_org_security_group_id}",
+  ]
+
+  site                                   = "org"
+  syslog_address                         = "${var.syslog_address_org}"
+  worker_ami                             = "${data.aws_ami.tfw.id}"
+  worker_asg_max_size                    = 20
+  worker_asg_min_size                    = 1
+  worker_asg_namespace                   = "Travis/org"
+  worker_asg_scale_in_threshold          = 80
+  worker_asg_scale_in_evaluation_periods = 3
+  worker_asg_scale_in_period             = 300
+  worker_asg_scale_out_threshold         = 60
+  worker_asg_scale_out_qty               = 2
+  worker_config                          = "${data.template_file.worker_config_org.rendered}"
+  worker_docker_image_android            = "${var.amethyst_image}"
+  worker_docker_image_default            = "${var.garnet_image}"
+  worker_docker_image_erlang             = "${var.amethyst_image}"
+  worker_docker_image_go                 = "${var.garnet_image}"
+  worker_docker_image_haskell            = "${var.amethyst_image}"
+  worker_docker_image_jvm                = "${var.garnet_image}"
+  worker_docker_image_node_js            = "${var.garnet_image}"
+  worker_docker_image_perl               = "${var.amethyst_image}"
+  worker_docker_image_php                = "${var.garnet_image}"
+  worker_docker_image_python             = "${var.garnet_image}"
+  worker_docker_image_ruby               = "${var.garnet_image}"
+  worker_instance_type                   = "c3.8xlarge"
+  worker_queue                           = "enigma"
+
+  worker_subnets = [
+    "${data.terraform_remote_state.vpc.workers_org_subnet_1b2_id}",
+    "${data.terraform_remote_state.vpc.workers_org_subnet_1b_id}",
+    "${data.terraform_remote_state.vpc.workers_org_subnet_1e2_id}",
+    "${data.terraform_remote_state.vpc.workers_org_subnet_1e_id}",
+  ]
+}
+
+resource "null_resource" "language_mapping_json" {
+  triggers {
+    amethyst_image = "${var.amethyst_image}"
+    garnet_image   = "${var.garnet_image}"
+  }
+
+  provisioner "local-exec" {
+    command = <<EOF
+exec ${path.module}/../bin/format-images-json \
+  ${var.amethyst_image}=android,erlang,haskell,perl \
+  ${var.garnet_image}=default,go,jvm,node_js,php,python,ruby \
+  >${path.module}/generated-language-mapping.json
+EOF
+  }
+}
```

Comment:
This `null_resource.language_mapping_json` can be removed, as we only use the one generated in `aws-production-2` :+1: